releasenotes/notes/0.25/discrete-pulse-library-deprecation-3a95eba7e29d8d49.yaml

```diff
@@ -1,14 +1,16 @@
 ---
-features:
+features_pulse:
   - |
-    The :class:`~qiskit.pulse.SymbolicPulse` library was extended. The new pulses in the library are:
+
+    The :class:`~qiskit.pulse.SymbolicPulse` library was extended. The new pulse functions
+    in the library are::
 
       * :func:`~qiskit.pulse.library.GaussianDeriv`
       * :func:`~qiskit.pulse.library.Sech`
       * :func:`~qiskit.pulse.library.SechDeriv`
       * :func:`~qiskit.pulse.library.Square`
 
-    The new functions return a :class:`ScalableSymbolicPulse`, and match the functionality
+    The new functions return a :class:`~qiskit.pulse.ScalableSymbolicPulse` instance, and match the functionality
     of the corresponding functions in the discrete pulse library, with the exception of
-    `Square()` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
-    discrete `square()` where such a shift was induced by a :math:`\\pi` phase).
+    ``Square()`` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
+    discrete ``square()`` where such a shift was induced by a :math:`\\pi` phase).
```

Comment:
Just a question, not a comment: I thought in `math` mode you don't have to escape backslashes (at least I've never used it I think) -- or is this special for `\pi`?

Reply:
Yeah, it should need an escape here. You would normally need to do it in a python docstring (because python will try to use it as an escape sequence) but for a release note we don't have to worry about that.